src/convertToObject.js

```diff
@@ -7,6 +7,22 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  const splitString = sourceString
+    .split(';')
+    .filter((string) => string.includes(':'));
+  const result = {};
```

Comment:
[CODE STYLE] - Avoid variable namings like `result`. It will decrease code readability. Instead, use a descriptive name that reflects the purpose of the variable, such as `styleObject` or `cssStyles`.